lib/bpk-component-banner-alert/stories.js

```diff
@@ -399,6 +399,18 @@ storiesOf('bpk-component-banner-alert', module)
 
     return (
       <View>
+        <BpkBannerAlert
+          style={styles.bannerAlert}
+          type={ALERT_TYPES.neutral}
+          message={
+            <>
+              Your booking was successful. Your booking reference is{' '}
+              <BpkText textStyle="sm" weight="emphasized">
+                fr33sh4v0c4d0
```

Comment:
omg you used my example absolute madlad

Reply:
I loved it

Reply:
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-08-27 at 09 34 54](https://user-images.githubusercontent.com/30267516/91417393-9a2ab180-e848-11ea-8921-5cec9cfefebd.png)

Reply:
get down to del taco

Reply:
You love to see it